### DISCUSSION TITLE
wyze V3 with POE and RTSP streaming will occasionally drop out and require power cycle

### DISCUSSION THREAD

**Discussion by ericmagaha** [gtxaspec/wz_mini_hacks#751] [Q&A]
Hello,

I have a few V3 cams setup as follows:
1. Firmware 4.36.9.139
2. Name brand 32Gb SD Card formatted as FAT32
3. Uctronics POE adapters https://www.amazon.com/dp/B0B9ZLVGJ4 and `ENABLE_USB_ETH="true"` and `ENABLE_USB_ETH_MODULE_AUTODETECT="true"` set in wz_mini.conf
4. Go2RTC enabled as the RTSP server with the defaults for video settings in wz_mini.conf
5. NFS mount so the camera is writing MP4 files to a computer/hard drive on the same network.
6. Using Synology Surveillance Station on a local Synology NAS to record the camera RTSP stream.

Everything works decently well for a while (hour/day/few days).  Occasionally ONE (same/different) of the cams will just drop off the network, ALL of the following will happen:
1. Not available in the Wyze App.
2. Cannot ping it i.e. unreachable.
3. Stops writing to the NFS mount.
4. Shows as offline in Synology Surveillance Station.

Only way to get it back is to reboot it i.e. unplug it from the POE switch, or reboot the POE switch.  Then it comes back up just fine.

Some time later (hour/day/few days) one of the cams will drop (same/different).  Same symptoms.  Same process to get it back online.

Any thoughts or suggestions?  TIA.

**Comment by miovo** [comment]
Facing the same issue.
I have 8 V3 cams and all powered and connected with the Uctronics adapter. I can't see to find anything else on this issue but glad to see I'm not the only one... Are you still having this issue?

Can't seem to find any other adapter that will do power and data in one.

**Comment by ericmagaha** [comment]
Yep, still having the issue.  The Wyze V3 cams with the wz_mini_hacks and the Uctronics POE adapters have become a maintenance nightmare.
I have already replaced 3 of them with standard IP/POE cameras.  I have been consuming the RTSP with my Synology NAS and an FFMPEG process on a Raspberry PI which outputs MP4 files just like the Wyze do i.e. date/hour/minute.mp4.  And I can run Go2RTC on my Synology and create streams and Homekit so they are viewable on my Apple TV.
My guess is something to do with the Uctronics POE adapter.  I have 2 other Wyze V3 that are on hardwired ethernet adapters but they are not POE i.e. they still require power.  They are not having this issue.

**Reply by miovo** [reply]
Which non poe ethernet adapters are you used now for the Wyze cameras?

**Comment by ericmagaha** [comment]
I have had success with these.
https://www.amazon.com/gp/product/B01N1X28F1

**Comment by tismofied** [comment]
This is exactly the issue I am having with the UCTRONICS adapters. @ericmagaha did you managed to get them going or did you just moved on to something else?
I am in the same situation and these adapters have become a headache for me lol

**Comment by tismofied** [comment]
> I have had success with these.
> 
> https://www.amazon.com/gp/product/B01N1X28F1

What's the uptime on the cameras with these adapters? Would you still recommend them?

**Reply by ericmagaha** [reply]
I moved on from the Wyze cameras about 6 months ago and replaced them all with hardwired POE cameras and everything has been rock solid.

**Comment by premiumtek** [comment]
Chiming in to say same in some cases - 5 Wyze v3's with Unitronics POE adapters, I have two that are solid, other three have intermittent issues. I suspect disabling the firmware/firmware spoofing / and/or the POE adapters as the cause, although I don't have solid evidence. I did have all the same cams running with the same setup before Wyze pushed required FW updates and had no hiccups for 1-2yrs. Other weird but maybe related issue - I did have one Unitronic adapter that had a conflicting / same MAC as another, causing one or the other to drop intermittently.
